app/views/layouts/_footer.html.haml

```diff
@@ -4,43 +4,73 @@
       .pure-g
         .pure-u-1.pure-u-sm-1-2.pure-u-md-1-3.pure-u-lg-1-6
           %h4 About
-          =link_to "Exercism: Personal Edition", "https://exercism.io"
-          =link_to "Exercism: Teams Edition", "https://teams.exercism.io"
-          =link_to "About Exercism", about_page_path
-          =link_to "Our values", values_page_path
-          =link_to "Our team", team_page_path
+          %ul
+            %li
```

Comment:
Could you inline the `=` on these pls, so it's `%li= link_to...`

Reply:
Done

Reply:
Actually, that didn't work because it was just showing the raw output rather than the actual link.

Reply:
I don't think you've pushed :)

Reply:
You need to make sure there's not a space between the `%li` and `=`

Reply:
I got that approach to work for all of the lists except the social icons section because that contains extra nesting.

Reply:
Yeah. You'll need to nest those. That's fine.

Can you change it from 
```
%li=link_to 
```

to 
```
%li= link_to 
```

please.

Reply:
Done